jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationResources.kt

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation
+
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient
+import software.amazon.awssdk.services.cloudformation.model.StackStatus
+import software.aws.toolkits.jetbrains.core.ClientBackedCachedResource
+import software.aws.toolkits.jetbrains.core.Resource
+
+object CloudFormationResources {
+    @JvmField
+    val ACTIVE_STACK_SUMMARIES = ClientBackedCachedResource(CloudFormationClient::class, "cfn.active_stacks") {
+        listStackSummariesFilter { it.stackStatus() != StackStatus.DELETE_COMPLETE }
```

Comment:
Do we listStackSummariesFilter anywhere else? Not sure if really needs to exist

Reply:
fair